src/backend/expungeservice/expunger/analyzers/time_analyzer.py

```diff
@@ -133,6 +135,30 @@ def _mark_as_time_ineligible(charges: List[Charge], reason: str, eligibility_dat
         for charge in charges:
             charge.set_time_ineligible(reason, eligibility_date)
 
+    @staticmethod
+    def _set_arrests_as_eligible(charges_with_summary: ChargesWithSummary):
+        # If most recent conviction is time eligible
+        # AND all other charges in the same case are arrests
+        # Then make all other charges in the case time eligible
+        mrc = charges_with_summary.most_recent_conviction
```

Comment:
Might as well extract out 

`second_mrc = charges_with_summary.second_most_recent_conviction`

to keep it consistent with `mrc`.